ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -289,69 +324,70 @@ protected void doInBackgroundGuarded(Void... params) {
       // setting a limit at all), but it works because this specific ContentProvider is backed by
       // an SQLite DB and forwards parameters to it without doing any parsing / validation.
       try {
-        Cursor photos = resolver.query(
-            Images.Media.EXTERNAL_CONTENT_URI,
+        Cursor medias = resolver.query(
```

Comment:
change to `media`. `Medias` is a wrong phrase.

Reply:
Same. Why ```media```? We get multiple items